### DISCUSSION TITLE
Problem with create new project

### DISCUSSION THREAD

**Discussion by mvidal90** [pavjacko/renative#860] [General]
Hi! I want to create a new project. I run `rvn new`, when it is executing npx yarn add renative-template-hello-world@0.35.4, it answer this:

[ error ] [new] npx yarn add renative-template-hello-world@0.35.4 : FAILED. this could happen if you have package.json accidentally created somewhere in parent directory
[ task ] <= new

 $ rnv new - Done! 🚀
│  🚀  SUMMARY | 5/30/2022, 3:38:42 PM                                         
│  $ rnv new   
│  ReNative Version: 0.35.4                                                    
│  Engine: engine-core                                                         
│  Build Scheme (-s): debug                                                    
│  Env Info: darwin | x64 | node v16.13.0                                      
│  Executed Time: 0h:0m:29s:370ms                                              
│                                                                              
│ [ error ] [new] npx yarn add renative-template-hello-world@0.35.4 : FAILED. this could happen if you have package.json accidentally created somewhere in parent directory

And there is no package.json file.

Thank you!

**Comment by mihaiblaga89** [comment]
`FAILED. this could happen if you have package.json accidentally created somewhere in parent directory`
it looks like you are in a folder that is inside of another package. That's why you don't see a package.json because it already has one, in some parent folder. Switch to another directory that is not in a package or maybe you have a package.json in your home folder by mistake or something.

**Reply by mvidal90** [reply]
Hi @mihaiblaga89 ! Thank you for your answer.. That's right, I had a package.json in my user folder, and I didn't know it! Thank you!

**Comment by pauliusguzas** [comment]
@mvidal90 I see this is already solved, therefore closing, feel free to reopen
